pkg/apis/servicecatalog/doc.go

```diff
@@ -13,5 +13,6 @@ limitations under the License.
 
 // +k8s:defaulter-gen=TypeMeta
 // +k8s:deepcopy-gen=package,register
+// +k8s:conversion-gen=github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog
```

Comment:
This shouldn't be necessary on the internal API -- none of the internal APIs seem to have it in k8s.